gsi.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+  <include name="base-q.xml"/>
+  <remote name="phh" fetch="https://github.com/phhusson/" />
```

Comment:
this is stuff we should at least have a fork of and pull from

Reply:
In my opinion, I think we just need folk `device_phh_treble` from [phh](https://github.com/phhusson/device_phh_treble).
Others are some stable dependencies that we can not folk

Reply:
@ittat it's `fork` :). BTW having a fork even for stable dependencies ensuresq resilience:
 - if the original repo disappears
 - if the branch we use disappears

Reply:
OK